saleor/search/documents.py

```diff
@@ -67,12 +71,34 @@ class Meta:
 @orders.doc_type
 class OrderDocument(DocType):
     user = fields.StringField(analyzer=email_analyzer)
+    first_name = fields.StringField()
+    last_name = fields.StringField()
 
     def prepare_user(self, instance):
         if instance.user:
             return instance.user.email
         return instance.user_email
 
+    def prepare_first_name(self, instance):
+        user = instance.user
+        if user:
+            if user.first_name:
```

Comment:
It looks like a copy of the `prepare_first_name` function from the above,  can we move it into the util, replace in both functions and test? Same with the function below